File: C++/412.cpp
__________________________________________________________________________________________________
sample 8 ms submission
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> res(n,"");
        int flag;
        for(int i = 0;i<n;i++){
            flag = 0;
            int j = i+1;
            if(j%3 == 0){
                res[i] += "Fizz";
                flag = 1;
            }
            if(j%5 == 0){
                res[i] += "Buzz";
                flag = 1;
            }
            if(flag == 0)
                res[i] = to_string(j);    
        }
        return res;
    }
};
__________________________________________________________________________________________________
sample 9680 kb submission
class Solution {
public:
    vector<string> fizzBuzz(int n) {
        vector<string> ret(n);
        for(int i = 1; i <=n;++i){
            if(i%3){
                if(i%5){
                    ret[i-1]=to_string(i);
                }
            }
            else{
                ret[i-1]=string("Fizz");
            }
            if(i%5==0)ret[i-1]+=string("Buzz");
        } 
        return ret;
    }
};
__________________________________________________________________________________________________

File: Python3/412.py
__________________________________________________________________________________________________
sample 32 ms submission
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        res =  []
        
        
        for x in range(1,n+1):
            if  not x%3 and not x%5:
                res.append('FizzBuzz')
            elif not x%3:
                res.append('Fizz')
            elif not x%5:
                res.append('Buzz')
            else:
                res.append(str(x))
                
        return res
__________________________________________________________________________________________________
sample 13908 kb submission
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        ans = []
        for i in range(1, n + 1):
            if i % 15 == 0:
                ans.append("FizzBuzz")
            elif i % 5 == 0:
                ans.append("Buzz")
            elif i % 3 == 0:
                ans.append("Fizz")
            else:
                ans.append(str(i))
        return ans
__________________________________________________________________________________________________
